Home tascs 1 3

DimaDim4 opened the pull request.

DimaDim4 committed:
hv1 folder includes first version of CV page.

DimaDim4 committed:
hv2 folder contains an updated version of CV page and table page.

DimaDim4 committed:
hv3 folder contains promo page and 3 forms.

DimaDim4 committed:
Delete .gitignore

fo0lman commented on hv1_my_cv/index.html:
```diff
@@ -0,0 +1,80 @@
+    </main>
+</body>
+
+</html>
```
во всех файлах сделать последнюю строку пустой, гх ругается
![image](https://user-images.githubusercontent.com/9566996/152837729-434e1cdc-8fc4-46f4-86f3-a6ee1b20488b.png)

fo0lman reviewed (changes requested):
По форматированию не к чему придраться))) 
Ну тогда зачада найди то что хочешь чтобы поправить...
1й коммит добавить пустые строки
2й что хочешь, поправь/измени, на твоё усмотрение

DimaDim4 committed:
blank rows have been added at the end for files:
hv1_my_cv/index.html; hv2_cv_table/my_cv/index.html;  hv2_cv_table/table/table.html; hv3_my_form/index.html; hv3_my_form/pages/authorization.html;  hv3_my_form/pages/registration.html;  hv3_my_form/pages/search.html

DimaDim4 committed:
Item "rafik" has been added to the model select list.

fo0lman reviewed (approved):
Всё ок

fo0lman commented:
Теперь 
* main
git fetch
git pull
